README.md

```diff
@@ -15,6 +15,12 @@ $ npm install
 $ sed -i 's/mostPackage/my-awesome-package/g' package.json
 ```
 
+or one-line install:
+
+```
+bash <(curl -sSL https://git.io/vr1HI)
```

Comment:
Another alternative is: `curl -sSL https://git.io/vr1HI | bash`.  I'm fine with either, just wanted to see what you think.